build-tools/packages/build-cli/src/commands/generate/typetests.ts

```diff
@@ -53,9 +54,12 @@ export default class GenerateTypetestsCommand extends PackageCommand<
 	static readonly description = "Generates type tests for a package or group of packages.";
 
 	static readonly flags = {
-		level: Flags.custom<ApiLevel>({
-			description: "What API level to generate tests for.",
-			default: ApiLevel.internal,
+		entrypoint: Flags.custom<ApiLevel>({
+			// Temporary alias for back-compat
+			aliases: ["level"],
+			deprecateAliases: true,
+			description:
+				'What entrypoint to generate tests for. Use "public" for the default entrypoint. If this flag is provided it will override the typeValidation.entrypoint setting in the package\'s package.json.',
```

Comment:
Good job noting which takes priority when both are used. Also makes the other way to set it discoverable!